tools/UpdatePaths/Scripts/71507_tablet_to_pda.txt

```diff
@@ -0,0 +1,6 @@
+# updates pda subtypes to their new type, and deletes base pdas as they were removed.
+
+/obj/item/modular_computer/tablet/pda : @DELETE
```

Comment:
Not /obj/item/modular_computer/pda?

Reply:
oh right i wanted to delete base tablets, my bad